test-suite/testheaders.sh

```diff
@@ -41,7 +41,7 @@ EOF
             echo "Fail."
             exitCode=1
         fi
-        rm $t.cc $t.o
+        rm -f $t.cc $t.o
```

Comment:
> "make check" hangs on a request to confirm removal of a read-only file temporary file.

What makes a temporary file read-only?

Reply:
Automake internals. I haven't investigated the causes, only the symptoms, and this whole script is going to be removed as part of PR #1463